provider/backup_utils.py

```diff
@@ -285,7 +285,7 @@ def blockdev_mirror_nowait(vm, source, target, **extra_options):
 
 @fail_on
 def blockdev_mirror(vm, source, target, **extra_options):
-    timeout = int(extra_options.pop("timeout", 600))
+    timeout = int(extra_options.pop("timeout", 900))
```

Comment:
hi @aliang123 , as the timeout issue only happened with 512 disk, is this a configuration issue? I mean if you have some cases testing on 512 disk, then add timeout=900 for those cases, no need to change the code here

Reply:
@zhencliu, yes, you are right. At present, all the failed cases were all 512 cluster_size related. So set the timeout there is enough. Thanks for pointing it out.